src/handlebars/lastlayer.cc.hbs

```diff
@@ -2,6 +2,12 @@
 #include "lastlayer.h"
 #include <cassert>
 
+#ifdef __enable_vcd_dump
```

Comment:
Lets follow this [convention](https://google.github.io/styleguide/cppguide.html#The__define_Guard) on `#define` guards. I will modify `lastlayer.h` after we merge this to follow the same convention.

Reply:
Hi Luis, are you looking for just a macro name change or actually using the "ifndef, define, endif" structure? This isn't really a header guard so I'm not really sure how to use the standard "ifndef, define, endif" semantics.

Reply:
Just macro name change is fine. I was thinking on something like `LASTLAYER_VCD`